python/py-nameutils/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-nameutils
+version             1.0.0
+revision            0
+
+license             GPL-3+
+platforms           any
```

Comment:
this should be "{darwin any}" as files will be installed in a different location on non-Darwin OSes (see other Python ports where this is set)

Reply:
Sure. I see lots of both "any" and "{darwin any}" (about half and half).

Reply:
yeah, all the `*_select` ports are `{any}` but Python ports should be `{darwin any}` for the reason I mentioned above. See [here](https://lists.macports.org/pipermail/macports-dev/2022-October/044765.html) for Joshua's explanation on this topic.

Reply:
Thanks. I've just had a look and there are 3 python ports that have `any`, and 1 that has `{darwin all}`, and 319 that default to `darwin` (but maybe they all include compiled code).

    2077 python/py-*/Portfile
    319 without platforms (i.e. default to darwin)
    1758 with platforms
    1440 with platforms {darwin any}
    308 with platforms darwin (or darwin freebsd or darwin linux or {darwin >= 11} etc.)
    1 with platforms {darwin all}
      py-pylint ?
    6 with platforms any
      py-bootstrap-modules ?
      py-cairosvg_select OK
      py-gpilab-examples ?
      py-html2text_select OK
      py-photomosaic ?
      py-pss_select OK
    3 with platforms macosx
      py-pyqt4
      py-pyqwt
      py-sip4